meta-lmp-base/recipes-support/resize-helper/resize-helper/resize-helper

```diff
@@ -38,8 +38,16 @@ RESIZE2FS=$(which resize2fs) || { echo "E: You must have resize2fs" && exit 1; }
 
 # find root device
 ROOT_DEVICE=$(findmnt --noheadings --output=SOURCE / | cut -d'[' -f1)
+# identify a possible mapper-based device (e.g. luks)
+if echo ${ROOT_DEVICE} | grep -q "^/dev/mapper/"; then
+	DM_NAME=`basename ${ROOT_DEVICE}`
+fi
 # prune root device (for example UUID)
 ROOT_DEVICE=$(realpath ${ROOT_DEVICE})
+# check if root device is available via device-mapper / luks
+if basename ${ROOT_DEVICE} | grep -q "^dm-"; then
```

Comment:
We can use the previous DM_NAME to check if this is in a device mapper

```
if [ -n "${DM_NAME}" ] ; then
```

Reply:
Done.